docs/guide/docs/integrating/existing-backend.md

```diff
@@ -1,26 +1,27 @@
 ---
 id: existing-backend
-title: Connecting your bot with your existing backend
+title: Connecting An Existing Backend
 ---
+![This feature is available to Botpress Enterprise license holders.](../assets/botpress-enterprise-feature.png)
 
-Botpress makes it easy to communicate data securely between your backend and your bot using JWT. Store any data you'd like in the token and pass it to the webchat using the `externalAuthToken` configuration option.
+Botpress makes it easy to transmit data securely between your backend and your chatbotusing JWT. Store any data you'd like in the token and pass it to the web chat using the `externalAuthToken` configuration option.
```

Comment:
`your backend and your chatbotusing`. Probably the have a space between `chatbot using` 😅